modules/search/instant-search/components/search-result-minimal.jsx

```diff
@@ -195,7 +137,11 @@ class SearchResultMinimal extends Component {
 					} ) }
 				</span>
 				<h3>
-					{ this.renderPostTypeIcon() }
+					<PostTypeIcon
+						postType={ fields.post_type }
+						shortcodeTypes={ fields.shortcode_types }
+						imageCount={ fields[ 'has.image' ] }
```

Comment:
Just confirming: is `fields[ 'has.image' ]` numerical?

Reply:
It is indeed:

<img width="425" alt="Screen Shot 2019-10-22 at 09 57 51" src="https://user-images.githubusercontent.com/17325/67242652-74a77b80-f4b2-11e9-8444-3bea454a028c.png">